src/shared/styles/theme.css

```
@@ -14,6 +14,44 @@
+  --color-beige-30: #605D59;
+  --color-beige-40: #74716C;
+  --color-beige-50: #9D9A95;
+  --color-beige-60: #BBB8B2;
+  --color-beige-70: #DFDCD6;
+  --color-beige-80: #EDEAE4;
+  --color-beige-90: #F3EFEA;
+  --color-beige-95: #F8F5F0;
+  --color-beige-99: #FFFCF9;
+
+
+  --opacity-0: #00000000;
```

Stylelint (17.2.0)

[error] Expected "#00000000" to be "#0000" (color-hex-length)

(color-hex-length)